### ISSUE TITLE
samplv1 lv2 hangs reaper when loading

### ISSUE THREAD

**Issue by JRSV** [rncbc/samplv1#40]
Hello

I am having a very strange issue with samplv1 in Ubuntu 22.04 and Reaper. Earlier today I opened and loaded the sampler with no issue, however during usage, it suddenly delayed all midi messages and playback was done after several seconds... (10 or maybe 20), then it crashed Reaper. Now when I open Reaper and try to load the sampler it hangs Reaper and all the UI. attached a screenshot. After this, Reaper, the sampler, and everything related just freezes. 
![sampler-freeze](https://github.com/rncbc/samplv1/assets/18346930/9cf8b23f-4a38-41ab-901d-28adb27aec57)

p.s. Running Reaper in the terminal does not spit out any information

any help will be appreciated

**Comment by rncbc** [comment]
so, it was working before, and now it doesn't? 

what happened? maybe a system reset would help? ;)

**Comment by JRSV** [comment]
yes, it worked for 3 minutes, then I had no midi response, 20 seconds later all the notes which I thought had no response started to trigger the samples (as normal but with huge latency). I did restart the machine, is this what you mean from  "system reset". In ardour it works with no issue. But I do wonder what happened.
